### PR TITLE
Fix Network Manager connection names for Leap micro

### PR DESCRIPTION
**Pull request by Bischoff**
## What does this PR change?

This PR changes "Network Connection" into "Network connection 1" and "Network connection 2".

It also installs bind-utils, needed for "host" utility by test suite.

### CODE REVIEW

File: backend_modules/libvirt/host/combustion [hunk 1]
```diff
@@ -64,7 +72,7 @@ PACKAGES="$PACKAGES salt-minion"
 %{ endif }
 
 %{ if testsuite }
-PACKAGES="$PACKAGES andromeda-dummy milkyway-dummy virgo-dummy timezone expect"
+PACKAGES="$PACKAGES andromeda-dummy milkyway-dummy virgo-dummy timezone expect bind-utils"
```
Comment (cbosdo) [inline backend_modules/libvirt/host/combustion:75]: `bind-utils` is not installed on SLE Micro, would we need it there too?

**Comment by Bischoff** [comment]
> AFAIK this is combustion only. ID we need similar things for cloud-init as well?

it's for combustion only

File: backend_modules/libvirt/host/combustion (hunk 1 above)
Comment (Bischoff) [reply backend_modules/libvirt/host/combustion:75]: SLE micro uses cloud-init

i can see it installed:
```
suma-head-pxy:~ # rpm -qf /usr/bin/host
bind-utils-9.16.44-150500.8.12.2.x86_64

suma-head-pxy:~ # grep PRETTY /etc/os-release 
PRETTY_NAME="SUSE Linux Enterprise Micro 5.5"
```

**Comment by Bischoff** [comment]
Current version seems to work, merging.

Fabian Vogt seems to have a better and simpler solution, we can always apply it later on.

**Pull request merged**